### ISSUE TITLE
Make xlsx an optional dependency

### ISSUE THREAD

**Issue by quentincaffeino** [nhn/tui.grid#1491]
## Current Behavior

Bundle size increased almost 4 times when xlsx was added. Not all projects require this functionality.
Before: https://bundlephobia.com/package/tui-grid@4.19.0 (108.5 kB)
After: https://bundlephobia.com/package/tui-grid@4.19.1 (428.9 kB)

PR that added this functionality: https://github.com/nhn/tui.grid/pull/1465

## Expected Behavior

It would be nice to have this as an optional feature.

**Comment by jajugoguma** [comment]
@quentincaffeino 
Sorry for late replying.
`xlsx` is set as an external dependency so there is no change to the actual bundle size as shown in the image below.
Bundlephobia seems to include all of these external dependencies as well to determine the size of the bundle.
I will check further, and if you have any other comments, please leave a comment.
<img width="427" src="https://user-images.githubusercontent.com/41339744/139625442-12e16169-45c7-476f-9574-b11e47ff7a1b.png">

**Comment by quentincaffeino** [comment]
@jajugoguma, I guess bundlephobia has this issue because it is included as a dependency in `package.json`. Shouldn't `xlsx` be listed as a `peerDependency`?
